deploy/osd-22018-remove-dns-tolerations/04-job.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "sre-remove-dns-tolerations"
+  namespace: "openshift-dns"
```

Comment:
Do we need to run in this namespace?

Maybe out of topic, but I think we do not have a good way to clean up those sa/job/roles, etc. If we have convention to put those temporary one-off resources in a single place, it would be easy for us to recycle?

Reply:
I also am not the most happy putting it to openshift-dns, but it seemed to be the most appropriate one.

I check with `oc get ns` but didn't really see a decent namespace - it could probably go to `openshift-sre-pruning`?

Reply:
I have no better idea, just as I said above it might be out of topic. maybe something we can figure out later on.

Reply:
I've changed to `openshift-sre-pruning` for now so it doesn't clutter a namespace that is more common to be checked by other engineers / customers.